docs/what-is-zarf.md

```diff
@@ -0,0 +1,225 @@
+
+Can you make a direct network connection? Even better.
+
+Consider the art of the possible and use what you can&mdash;Zarf will continue to work, regardless.
+
+&nbsp;
+
+
+### Appliance Mode
+
+<a href="../.images/what-is-zarf/appliance-mode.png">
+  <img alt="how it works" src="../.images/what-is-zarf/appliance-mode.png" height="295" />
+</a>
+
+#### (0) - Package
+
+Move a Zarf release + your desired packages to your Zarf cluster machine.
+
+#### (1) - Zarf cluster
+
+Make your machine into a **single node** Zarf cluster with the command: `zarf init`.
+
+Recommended Zarf components: `management`.
+
+Deploy your package into the Zarf cluster with the command: `zarf deploy`.
+
+#### (✓) - Use it
+
+Connect directly to the Zarf cluster to access your newly unpackaged applications!
+
+&nbsp;
+
+
+### Utility Mode
+
+<a href="../.images/what-is-zarf/utility-mode.png">
+  <img alt="how it works" src="../.images/what-is-zarf/utility-mode.png" height="283" />
+</a>
+
+#### (0) - Package
+
+Move a Zarf release + desired packages to your Zarf cluster machine.
+
+#### (1) - Zarf cluster
+
+Make your Zarf cluster machine into a **single node** Zarf cluster with the command: `zarf init`.
+
+Recommended Zarf components: `management` & `gitops-services`.
+
+Deploy your package into the Zarf cluster with the command: `zarf deploy`.
+
+#### (2) - Downstream cluster
+
+Point your downstream cluster to the Zarf cluster **source code** & **container image repository** for access to base software & updates (hint: [GitOps](https://www.weave.works/technologies/gitops/)).
+
+#### (✓) - Use it
+
+Once your downstream cluster has settled accounts with the Zarf cluster, connect to the downstream cluster to access your newly unpackaged applications!
+
+&nbsp;
+
+
+## Target Use Cases
+
+Zarf's possibilities are endless&mdash;Zarf developers' time is not. Thus, scope definition is in order.
+
+Here are the things we think Zarf should get really good at, listed top-down in order of importance:
+
+- Make movement of software "across the air gap" an open source "solved problem".
+
+- Make it trivial to deploy & run Kubernetes apps "at the Edge".
+
+- Make it easy to support GitOps-based k8s cluster updates in isolated environments.
+
+- Make it possible to support GitOps-based k8s cluster updates in internet-connected-but-independent environments (think: dependency caching per availability zone, etc).
+
+&nbsp;
+
+
+## What can be packaged?
+
+To reiterate: Zarf's possibilities are endless&mdash;Zarf developers' time is not. Thus, scope definition is again in order.
+
+Given Zarf's being a "k8s cluster to serve _other_ k8s clusters", the following types of software can be rolled into a Zarf package:
+
+- container images &mdash; to serve images for the Zarf & downstream clusters to run containers from.
+
+- git repositories &mdash; to serve as the git-based "source of truth" for downstream "GitOps"ed k8s clusters to watch.
```

Comment:
now also any generic git repos needed from the online to offline side since the last big git PR merge.  Think "I'd love to have the docs for this cool tool in my airgap, can zarf bring those from github to here for me?"